composer.json

```diff
@@ -14,13 +14,18 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^9",
-        "dealerdirect/phpcodesniffer-composer-installer": "dev-master",
-        "phpstan/phpstan": "1.7.x",
+        "dealerdirect/phpcodesniffer-composer-installer": "v1.0.0",
```

Comment:
@aashwin-rvvup I would allow #7  to be reviewed  & merged first as `composer.json` is updated with flexible versioning (allows minor & patch changes but not major ones)